vms/platformvm/metrics/metrics.go

```diff
@@ -31,10 +31,10 @@ type Metrics interface {
 	// Mark that a validator set was cached.
 	IncValidatorSetsCached()
 	// Mark that we spent the given time computing validator diffs.
-	AddValidatorSetsDuration(time.Duration)
+	SetValidatorSetsDuration(time.Duration)
```

Comment:
@aaronbuchwald, @StephenButtolph  not sure why we cumulate time here. In order to extract single requests time series (duration and diff applied count) I'd have to take rate. Should we switch this to just Set*, so to track single requests data?